web-api/src/persistence/postgres/cases/getCasesByDocketNumbers.ts

```diff
@@ -14,37 +14,82 @@ import { fromKyselyCase } from '@web-api/persistence/postgres/cases/mapper';
 import { CaseKysely } from '@web-api/persistence/postgres/cases/schema';
 import { difference, isEmpty, partition, sortBy } from 'lodash';
 
-export async function getCasesByDocketNumbers({
+export const ALL_OMITTABLE_CASE_FIELDS = [
```

Comment:
We started down _including_ fields, which might be more intuitive, rather than _excluding_ fields. We didn't finish, and we think this is still a good improvement.